plugins/themes/default/styles/pages/catalog.less

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file  plugins/themes/default/styles/pages/catalog.less
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2003-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @brief Styles applying to catalog page
+ * @link templates/frontend/pages/catalog.tpl
+ */
+.page_catalog {
+	.pkp_series_nav_menu{
+		border-top: @bg-border;
+		> ul {
+			display: inline;
+			list-style: none;
+			padding: 0px;
+			> li {
+				display: inline;
+				&::after {
+			 		content: ", ";
```

Comment:
We can't put things like this into CSS because it needs to be translatable. I thnk there's a `list.separator` locale key or something like that we can use in the template.

Smarty has a `@last` syntax you can use to skip it on the last iteration of a foreach loop:  https://www.smarty.net/docs/en/language.function.foreach.tpl

Reply:
hmm sure I understand. It would of course be nice to have a formatting that allows you to change the style without adding a new custom template to the theme.

What if we do not add commas at all?

Reply:
ah but nevermind, I will add a style to the separator as well so that you can hide that with CSS if you want.

Reply:
:+1: (it should be inside each list item, i think, and add `aria-hidden="true"` to the separator so that screen readers don't read it)

Reply:
ok, added the aria-hidden rule!